docs/user-guide/petset.md

```diff
@@ -0,0 +1,278 @@
+
+```shell
+$ kubectl exec -it web-0 /bin/sh
+web-0 # apt-get update && apt-get install -y dnsutils
+...
+
+web-0 # nslookup -type=srv nginx.default
+Server:		10.0.0.10
+Address:	10.0.0.10#53
+
+nginx.default.svc.cluster.local	service = 10 50 0 web-1.ub.default.svc.cluster.local.
+nginx.default.svc.cluster.local	service = 10 50 0 web-0.ub.default.svc.cluster.local.
+```
+
+## Updating a Pet Set
+
+You cannot update any field of the PetSet except `spec.replicas`. You can update the replicas field using standard kubectl update commands like [patch](/docs/user-guide/kubectl/kubectl_patch) and [edit](/docs/user-guide/kubectl/kubectl_edit). Pet Set currently *does not* support image upgrade as noted in the section on [limitations](#alpha-limitations).
+
+## Scaling a Pet Set
+
+You can scale a Pet Set by updating the "replicas" field. Note however that the controller will only:
+
+1. Create one pet at a time, in order from {0..N-1}, and wait till each one is in [Running and Ready](/docs/user-guide/pod-states) before creating the next
+2. Delete one pet at a time, in reverse order from {N-1..0}, and wait till each one is completely shutdown (past its [terminationGracePeriodSeconds](/docs/user-guide/pods/index.md#termination-of-pods)) before deleting the next
+
+## Deleting a Pet Set
+
+Cleaning up a Pet Set is somewhat manual, as noted in the [limitations section](#alpha-limitations). You can delete a Pet Set using Kubectl, but this will *not* scale it down to 0:
+
+```shell
+$ kubectl delete -f petset.yaml
+petset "web" deleted
+
+$ kubectl get po -l app=nginx
+NAME      READY     STATUS    RESTARTS   AGE
+web-0     1/1       Running   0          21h
+web-1     1/1       Running   0          21h
+
+$ kubectl delete po -l app=nginx
+pod "web-0" deleted
+pod "web-1" deleted
+```
+
+Deleting the pods will *not* delete the volumes. Until we finalize the recycle policy for these volumes they will have to get cleaned up by an admin. This is to ensure that you have the chance to copy data off the volume before deleting it. Simply deleting the PVC after the pods have left the [terminating state](/docs/user-guide/pods/index.md#termination-of-pods) should trigger deletion of the backing Persistent Volumes.
+
+**Note: you will lose all your data once the PVC is deleted, do this with caution.**
+
+```shell
+$ kubectl get po -l app=nginx
+$ kubectl get pvc -l app=nginx
+NAME        STATUS    VOLUME                                     CAPACITY   ACCESSMODES   AGE
+www-web-0   Bound     pvc-62d271cd-3822-11e6-b1b7-42010af00002   0                        21h
+www-web-1   Bound     pvc-62d6750e-3822-11e6-b1b7-42010af00002   0                        21h
+
+$ kubectl delete pvc -l app=nginx
+$ kubectl get pv
+```
+
+If you simply want to clean everything:
+
+``` {% raw %}
+$ grace=$(k get po web-0 --template '{{.spec.terminationGracePeriodSeconds}}')
+$ kubectl delete petset,po -l app=nginx
+$ sleep $grace
+$ kubectl delete pvc -l app=nginx
+```
```

Comment:
Are you suggesting that I quote the quote like you have? i.e:
```
```shell{% raw %}
$ grace=$(k get po web-0 --template '{{.spec.terminationGracePeriodSeconds}}')
$ kubectl delete petset,po -l app=nginx
$ sleep $grace
$ kubectl delete pvc -l app=nginx
{% endraw %}```
```

Because I get 
```shell{% raw %}
$ grace=$(k get po web-0 --template '{{.spec.terminationGracePeriodSeconds}}')
$ kubectl delete petset,po -l app=nginx
$ sleep $grace
$ kubectl delete pvc -l app=nginx
{% endraw %}```

vs what i have now which is

```{% raw %}
$ grace=$(k get po web-0 --template '{{.spec.terminationGracePeriodSeconds}}')
$ kubectl delete petset,po -l app=nginx
$ sleep $grace
$ kubectl delete pvc -l app=nginx
```